FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2BottomSheetActivity.kt

```diff
@@ -429,14 +429,14 @@ private fun CreateActivityUI() {
                     style = ButtonStyle.Button,
                     size = ButtonSize.Medium,
                     text = "+ 8 dp",
-                    enabled = !hidden,
+                    enabled = hidden,
```

Comment:
not having context regarding this, can you tell me what is the reason for this change?

Reply:
so currently in demo app, peek height option is only shown when drawer activates
<img width="355" alt="image" src="https://github.com/user-attachments/assets/a90675b5-b51d-4691-bcd8-4ccced03f403">
and once the drawer activates, the option being unusable as it's hidden behind the scrim
so here we are making the change so that peek height option is enabled when the drawer is not activated so that we can configure the peek height before activating the drawer